expression/builtin_math_test.go

```diff
@@ -449,10 +449,10 @@ func (s *testEvaluatorSuite) TestDegrees(c *C) {
 		{float64(1), float64(57.29577951308232), false, false},
 		{float64(math.Pi), float64(180), false, false},
 		{float64(-math.Pi / 2), float64(-90), false, false},
-		{"", float64(0), false, true},
+		{"", float64(0), false, false},
 		{"-2", float64(-114.59155902616465), false, false},
-		{"abc", float64(0), false, true},
-		{"+1abc", float64(0), false, true},
+		{"abc", float64(0), false, false},
```

Comment:
from line 434 to line 438
set the `IgnoreTruncate` to false for this statement context,
so StrToFloat("abc") will not get a truncate error.

@youyouhu